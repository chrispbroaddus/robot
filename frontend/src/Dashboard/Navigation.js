import React from 'react'
import styled from 'styled-components'
import Logo from '../Shared/Logo'

const Navbar = styled.div`
  width: 100%;
  background-color: white;
  box-shadow: -1px 2px 8px 0 rgba(155, 155, 155, 0.2);
  padding: 20px 42px;
`

const Navigation = props => {
  return (
    <Navbar>
      <Logo />
    </Navbar>
  )
}

export default Navigation
