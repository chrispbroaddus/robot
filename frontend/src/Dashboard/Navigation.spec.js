import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Navigation from './Navigation'

describe('Navigation', () => {
  let props

  beforeEach(() => {
    props = {}
  })

  it('should render self and subcomponents', () => {
    const wrapper = shallow(<Navigation {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
