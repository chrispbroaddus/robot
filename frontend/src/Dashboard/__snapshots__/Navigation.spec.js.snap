// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should render self and subcomponents 1`] = `
<styled.div>
  <Logo
    className=""
    shadow={false}
  />
</styled.div>
`;
